resource_manager/unit_tests/test_project.py

```diff
@@ -291,6 +321,10 @@ def test_undelete_with_reload_data(self):
             'name': 'Project Name',
             'labels': {'env': 'prod'},
             'lifecycleState': 'DELETE_REQUESTED',
+            'parent': {
+                'type': 'organization',
+                'id': '433637338579'
```

Comment:
Great practice for trailing commas here (and every edited test that looks the same)